### DISCUSSION TITLE
Welcome to vsphere-automation-sdk-python Discussions!

### DISCUSSION THREAD

**Discussion by jobingeo** [vmware-archive/vsphere-automation-sdk-python#362] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shanewhite-bc** [comment]
Good afternoon.
I have a customer who wants to know if this SDK to be also published on pypi.org?
The reason is that there is no connectivity from their internal github server to github.com (they are a financial institution and thus very security focus). However, their artifactory server does have access to pypi.org
If publishing to pypi.org isn't possible, can you suggest another way of making this SDK available within this customer?

**Reply by jobingeo** [reply]
This SDK has several components (vCenter/vSphere, VMware Cloud, NSX etc). The vCenter component of this SDK is published in PyPi, starting vSphere 8.0.3.0 release, https://pypi.org/project/vmware-vcenter/

**Reply by shanewhite-bc** [reply]
Thanks. Have passed this on. 
Appreciate the fast response.

**Comment by pangdanyep** [comment]
Hello~
I have a question.Do you know how to get the library ovf template nework adapater details infomation and the disks infomation？  
QAQ~
